CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+not a dogma, on the other hand, we value a clean and consistent Git history. Please help us maintain this standard.
+
+Now you can push your local branch to your GitHub repository and make the pull request to the Percona repository.
+
+Please, write meaningful title for the PR and put the number of Jira issue at the beginning of the first line in the
+description. The second line is usually a link to the Jira issue. Then the short description follows. No need to write a
+long description because you wrote one in the Jira issue.
+
+Example of a pool request form:
+
+Title:
+```
+PS-9876 Fix of the compression bug
+```
+Descriptopn:
+```
+PS-9876 Fix of the compression bug
+
+https://perconadev.atlassian.net/browse/PS-9876
+
+The first commit is update of zlib, the second is the fixes of compression function calls.
+```
+
+### Peer review
+
+Peer review of the code always goes before PR merge. So, you have to add some core development team members in the "Reviewers" field of PR. Best guess is to assign the PR to Yura Sorokin, our team lead.
+
+Our GitHub pipeline runs a lot of checks on each PR. Please make sure you fixed all comments provided by checks. Well,
+sometime not all of them :) , you understand...
+
+Following this step, the development team will conduct a thorough review of your code at their earliest convenience. Our
+primary objective is to ensure the highest quality standards. We encourage you to interpret our comments as
+opportunities for improvement and constructive feedback.
+
+When the review is finished and all the issues are addressed, your pull request will be approved and merged into the main source tree.
+
+Congratulations! You are now a recognized contributor.
+
+## Your Code and MySQL Versions
+
+Percona officially supports 3 version lines of MySQL:
+
+1. 8.0.X - stable line.
+2. 8.4.X - next stable line, or LTS line.
+3. 9.0.X - innovation line, a test bed for everything new at the moment.
+
+Next LTS version may be 9.7, so until then 9.X are "innovation" versions.
+
+It is not easy to keep track of all the changes and understand how the new code will work with different versions.
+Therefore, your best bet is to work with the version line you're currently using and familiar with. Nevertheless,
+Percona developers team will take care of your code, and port or backport it to other version lines if it is applicable.
+
+## When will My Code be in the release?
+
+Percona releases follow upstream MySQL releases. We merge upstream code changes and our code from the "trunk" as soon as
+the next official upstream release becomes available. Then we build packages and test everything thoroughly. It takes
+time.
+
+We use a "train" approach. It means, that if some code is already in one of the "trunk" branches before the official
+upstream release, it will be released in Percona's distribution of new release. So, if we are already in the process of
```

Comment:
Confusing.
"it will be released in Percona's distribution of a new release" - we have a Distribution release - are you talking about that or what? What is a distribution of a new release?"

Reply:
Changed to "packages"